media/css/firefox/new/desktop/download.scss

```diff
@@ -8,6 +8,7 @@ $image-path: '/media/protocol/img';
 
 @import '../../../../protocol/css/components/emphasis-box';
 @import '../../../../protocol/css/components/modal';
+@import '../../../../protocol/css/components/section-heading';
```

Comment:
This page also has a prototype block component that should be able to be swapped out for split (not a blocker for merging though).

Reply:
hmm this is a little tricky with the inline SVG. I don't think it can use the macro, it would have to just be coded into the page. Which we can totally do. But I also think that hero image is going to change in the next few weeks for MR1 so maybe we can leave it for now and change to the component when we're doing the other updates.

Reply:
And the weird browser compat table thing could be as annoying as the SVG. Okay, this can go as-is. 👍